index.html

```diff
@@ -1,21 +1,152 @@
+          <hr class="header-line3" />
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="background1"></div>
+      <div class="main-content">
+        <div class="main-box-container">
+          <img
+            src="https://static.wixstatic.com/media/2e2a49_bd3a1599bc1b4265a01b2754b99b8536~mv2_d_5760_3840_s_4_2.jpeg/v1/crop/x_2703,y_43,w_2199,h_2205/fill/w_205,h_205,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/AdobeStock_230407429.jpeg"
+            alt="Maya Photo"
+          />
+          <div class="main-name">
+            Maya <br />
+            Nelson
+          </div>
+          <hr class="blue-line" />
+          <div class="main-title">PROJECT MANAGER</div>
+          <div class="contact">
+            <img
+              alt="Facebook"
+              src="https://static.wixstatic.com/media/0fdef751204647a3bbd7eaa2827ed4f9.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/0fdef751204647a3bbd7eaa2827ed4f9.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+            <img
+              alt="Twitter"
+              src="https://static.wixstatic.com/media/c7d035ba85f6486680c2facedecdcf4d.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/c7d035ba85f6486680c2facedecdcf4d.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+            <img
+              alt="LinkedIn"
+              src="https://static.wixstatic.com/media/6ea5b4a88f0b4f91945b40499aa0af00.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/6ea5b4a88f0b4f91945b40499aa0af00.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+            <img
+              alt="Instagram"
+              src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_25,h_25,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/01c3aff52f2a4dffa526d7a9843d46ea.png"
+              style="width: 25px; height: 25px; object-fit: cover"
+            />
+          </div>
+        </div>
+        <div class="hello-part">
+          <div class="hello-text">Hello</div>
+          <div class="role">Here's who I am & what I do</div>
+          <div class="btn">
+            <button class="resume-btn">RESUME</button>
+            <button class="projects-btn">PROJECTS</button>
+          </div>
+          <p class="p1">
+            I'm a paragraph. Click here to add your own text and edit me. It’s
+            easy. Just click “Edit Text” or double click me to add your own
+            content and make changes to the font.
+          </p>
+          <p class="p2">
+            I’m a great place for you to tell a story and let your users know a
+            little more about you.
+          </p>
+        </div>
+      </div>
+    </main>
+    <footer>
+      <p class="purpose1">
+        This website design was created by Wix.com, and is used here for
+        strictly educational purposes.
+      </p>
+      <div class="footer-call">
```

Comment:
The 3 contact elements (call, write, follow) are grouped together on larger screens. As we discussed on the call, it's ok to add a wrapper element around them to make it easier to achieve that styling.